styles/nav.scss

```diff
@@ -153,6 +153,32 @@ $daptiv-subnav-tab-h-pad: 14px;
         }
     }
 
+    .daptiv-subnav-prev-tab {
+        @extend .daptiv-subnav-tab;
+
+        &::before {
+            @extend .fa;
+            content: $fa-var-angle-double-left;
+            font-size: $icon-size-small;
+            margin-right: 3px;
+            position: relative;
+            top: 1px;
+        }
```

Comment:
these should be in a new section external to navigation: Iconography
ideally something like this
```sass
$mixin icon-small($position, $content) {
    &::$position {
        font-size: $icon-size-small;
        margin-right: 3px; /* probably create a variable for this */
        position: relative;
        top: 1px; /* probably create a variable for this  */
    }
}
.icon {
    @extend .fa;
    
    &.previous {
        =icon-small(before, $fa-var-angle-double-left);
    }
    &.next {
        =icon-small(after, $fa-var-angle-double-right);
    }
}
```

so then you would use it like this

```html
<a href="#prev" class="daptiv-subnav-tab icon previous">Previous</a>
<a href="#next" class="daptiv-subnav-tab icon next">Next</a>
```